specification/trafficmanager/resource-manager/readme.md

```diff
@@ -37,6 +37,39 @@ input-file:
 - Microsoft.Network/stable/2018-03-01/trafficmanager.json
 ```
 
+### Tag: package-2018-02
+
+These settings apply only when `--tag=package-2018-02` is specified on the command line.
+
+``` yaml $(tag) == 'package-2018-02'
+input-file:
+- Microsoft.Network/stable/2018-02-01/trafficmanager.json
+- Microsoft.Network/preview/2017-09-01-preview/trafficmanageranalytics.json
+
+# Needed when there is more than one input file
+override-info:
+  title: TrafficManagerManagementClient
+
+directive:
+  - suppress: R3023
+    reason: it's implemented in the main network spec
+    approved-by: "@fearthecowboy"

```

Comment:
@fearthecowboy Please confirm your approval for these suppressions

Reply:
Yes, these suppressions are ok. 

As far as I understand, they have things in the `Microsoft.Network` namespace, but it's still part of their RP.  (we've got a couple cases of this elsewhere)